### PR TITLE
Fix the deeplink for ios

### PR DESCRIPTION
**Pull request by DiegoVazquezNanini**
Revert original code for the deeplink.

### CODE REVIEW

**Comment by agulowaty** [comment]
how about some tests ?

File: src/main/scala/com/gu/mobile.notifications.client/legacy/NotificationBuilder.scala [hunk 1]
```diff
@@ -93,6 +93,11 @@ object NotificationBuilderImpl extends NotificationBuilder {
 
   private def buildAndroidBreakingNewsPayload(payload: BreakingNewsPayload, editions: Set[Edition]) = {
 
+    val androidLink = payload.link match {
+      case GuardianLinkDetails(contentApiId, _, _, _, _) => s"x-gu://www.guardian.co.uk/$contentApiId"
```
Comment (agulowaty) [inline src/main/scala/com/gu/mobile.notifications.client/legacy/NotificationBuilder.scala]: I'm missing something probably, but assuming x-gu is in-app link, why we're not consistent with the approach used in MAPI (for instance: https://github.com/guardian/mobile-apps-api/blob/1a4277fca954eb91a65ea8b2e9be4d19afe32394/common/app/lib/InAppUrlRewriter.scala#L21) ??

Comment (DiegoVazquezNanini) [inline src/main/scala/com/gu/mobile.notifications.client/legacy/NotificationBuilder.scala]: Hi @agulowaty 
Making the deep link consistent between MAPI and the Notification service is outside of scope of this PR. I am just reverting the behaviour of the deep link, though the change is supported on the current apps is not backward compatible with previous versions of the app. .
Please, raise a ticket if you feel strong about this inconsistency.

**Comment by agulowaty** [comment]
:+1:

**Pull request merged**